docs-devsite/firestore_.md

```diff
@@ -659,6 +687,11 @@ If the transaction completed successfully or was explicitly aborted (the `update
 > This API is provided as a preview for developers and may change based on feedback that we receive. Do not use this API in a production environment.
 > 
 
+> Warning: This API is now obsolete.
+> 
+> Instead of creating cache indexes manually, consider using `enablePersistentCacheIndexAutoCreation()` to let SDK decide whether to create cache indexes for queries running locally.
```

Comment:
Edit:

"Instead of creating cache indexes manually, consider using `enablePersistentCacheIndexAutoCreation()` to let the SDK decide whether to create cache indexes for queries running locally."

Reply:
And below....